src/Compilers/CSharp/Test/Syntax/Parsing/AwaitParsingTests.cs

```diff
@@ -1890,28 +1890,21 @@ public void BadLocalDeclarationAndAwaitExpressionInSyncContext()
                 N(SyntaxKind.Block);
                 {
                     N(SyntaxKind.OpenBraceToken);
-                    N(SyntaxKind.LocalDeclarationStatement);
+                    N(SyntaxKind.ExpressionStatement);
```

Comment:
this is a better parse than before.  `await goo(` should be a misplaced await-expression, not a delcaration of a variable `goo` with type `await`.